docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,269 @@
+
+```
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
+C:\Program Files (x86)\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+```
+
+On 32-bit Windows, the compiler tools are located here:
+
+```
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsc.exe
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsi.exe
+C:\Program Files\Microsoft SDKs\F#\4.0\Framework\v4.0\fsiAnyCpu.exe
+```
+
+You will need to add these to your PATH for Ionide to work.
+
+### macOS
+
+On macOS, Ionide uses Mono.  The easiest way to install Mono on macOS is via Homebrew.  Simply type the following into your terminal:
+
+```
+brew install mono
+```
+
+### Linux
+
+On Linux, Ionide also uses Mono.  If you're on Debian or Linux, you can use the following:
+
+```
+sudo apt-get update
+sudo apt-get install mono-complete fsharp
+```
+
+## Installing Visual Studio Code and the Ionide plugin
+
+You can install Visual Studio Code on the [code.visualstudio.com](https://code.visualstudio.com) website.  After that, there are a two ways to find the Ionide plugin:
+
+1. Use the Command Palette (ctrl-P on Windows, ⌘-P on macOS, ctrl-shift-P on Linux) and type the following:
+
+    ```
+    ext install Ionide
+    ```
+
+2. Click the Extensions icon and search for "Ionide":
+
+    ![](media/getting-started-vscode/vscode-ext.png)
+
+The only plugin required is **Ionide-fsharp**.  However, you can also install **Ionide-FAKE** and **Ionide-Paket** to get [FAKE](http://fsharp.github.io/FAKE/) and [Paket](https://fsprojects.github.io/Paket/) support.
+
+## Creating your first project Ionide
+
+To create a new F# project, open the Command Palette (ctrl-P on Windows, ⌘-P on macOS, ctrl-shift-P on Linux) and type the following:
+
+```
+>f#: New Project
+```
+
+You should see something like this:
+
+![](media/getting-started-vscode/vscode-new-proj.png)
+
+Select it by hitting **Enter**, which will take you to this step:
+
+![](media/getting-started-vscode/vscode-proj-type.png)
+
+This will select a template for a specific type of project.  There are quite a few options here, such as an FsLab template for Data Science or Suave template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach this step:
+
+![](media/getting-started-vscode/vscode-new-dir.png)
+
+Here you can choose to create the project in a specific directory.  If you leave it blank, it will choose the current directory Visual Studio Code is opened in.  This article uses the current directory.  Once you've made your decision, hit **Enter** to bring up the final stage:
+
+![](media/getting-started-vscode/vscode-proj-name.png)
+
+This lets you name your project.  F# uses PascalCase for project names.  This article uses `ClassLibrarySample` as the name.  Once you've entered the name you want for your project, hit **Enter**.
```

Comment:
PascalCase -> Pascal case

I feel you could simplify these instructions by being more direct. For example, here:
"Name your project `ClassLibrarySample` and hit **Enter**. F# uses Pascal case for project names. "